jobs/service-fabrik-broker/spec

```diff
@@ -252,6 +258,19 @@ properties:
   cf.password:
     description: "Admin password for the Cloud Foundry cloud controller"
 
+  cred_provider.credhub_url:
+    description: "Credhub API endpoint"
+  cred_provider.credhub_uaa_url:
+    description: "UAA endpoint"
+  cred_provider.credhub_client_id:
+    description: "UAA client id for generating OAUTH token"
```

Comment:
Same as in the other PR should not the parameter name use uaa_ instead of cred_

Reply:
Answered in BnR [PR](https://github.com/cloudfoundry-incubator/service-fabrik-backup-restore/pull/35#discussion_r196321935) 
It does have uaa, just that its also prefixed with credhub explicitly.